float32[] id
float32[] xpos
float32[] ypos
float32[] intensity
int32[] victims
int32[] status
